rest/handler.go

```diff
@@ -476,6 +476,11 @@ func (h *handler) readJSONInto(into interface{}) error {
 	return ReadJSONFromMIME(h.rq.Header, h.requestBody, into)
 }
 
+// readSanitizeJSONInto reads and sanitizes a JSON request body into DbConfig.
+func (h *handler) readSanitizeJSONInto(config *DbConfig) error {
+	return ReadSanitizeConfigJSON(h.rq.Header, h.requestBody, config)
+}
```

Comment:
This looks like an unnecessary abstraction layer - it's not really providing any value other than pulling out the header and body from the handler. 

Since ReadSanitizeConfigJSON doesn't have any other callers, is there a reason to not just move it all in here?  It doesn't naturally belong in multipart.go.

Reply:
I thought we need to align it with the existing readJSONInto implementation which is doing similar thing but it is generic and that is the reason I kept this inside multipart.go. But I see what you mean and it would be cleaner to move it all in here.